### DISCUSSION TITLE
Failed getting RSSI from lib-python

### DISCUSSION THREAD

**Discussion by scf-ku** [bitcraze/discussions#1548] [Q&A] [answered]
Hi,

I'm trying to use the example script "crazyflie-lib-python/examples/radio-test.py" with Crazyflie 2.1+ and Crazyradio2.0.
But for some reason it is not returning an rssi value for any channel even though the ack rates seem good.

Channel 80 ack_rate: 1.0 rssi average: nan rssi std: nan
Channel 81 ack_rate: 1.0 rssi average: nan rssi std: nan
Channel 82 ack_rate: 1.0 rssi average: nan rssi std: nan
Channel 83 ack_rate: 1.0 rssi average: nan rssi std: nan
Channel 84 ack_rate: 1.0 rssi average: nan rssi std: nan

After that I checked this issue(https://github.com/bitcraze/crazyflie-lib-python/issues/131) and follow the exactly same method
(clone https://github.com/bitcraze/crazyflie2-nrf-firmware compile and bootload the firmware with the following commands)
$ make clean
$ make BLE=0 CONFIG=-DRSSI_ACK_PACKET
$ make BLE=0 CONFIG=-DRSSI_ACK_PACKET cload

And now my problem has not fixed yet.
Is there any solution to this problem?

**Comment by gemenerik** [comment, marked as answer]
It seems the command line define doesn't register somehow. Instead, add the following lines to a file named `config.mk` in your `crazyflie2-nrf-firmware` directory:
```mk
BLE=0
CFLAGS += -DRSSI_ACK_PACKET=1
```
Then
```mk
make clean
make
make cload
```
